app/uk/gov/hmrc/hec/models/hecTaxCheck/CTLookupStatus.scala

```diff
@@ -14,7 +14,7 @@
  * limitations under the License.
  */
 
-package uk.gov.hmrc.hec.models
+package uk.gov.hmrc.hec.models.hecTaxCheck
 
 sealed trait CTLookupStatus extends Product with Serializable
```

Comment:
this isn't being stored in the BE database so I'm not sure that it belongs in the `hecTaxCheck` package

Reply:
yes, good spot, will move it back

Reply:
updated https://github.com/hmrc/hec/pull/54/commits/aa47933b2d812011e367018d0a077f329c4e5a24